tox.ini

```diff
@@ -16,6 +16,10 @@ setenv =
 requires =
     pip>=19.2.1
 
+[testenv:py37]
+requires =
+    venusian
```

Comment:
the better way to do this is to use tags on the `[testenv]`:

```ini
[testenv]
command = ...
requires =
    pip>=19.2.1
    py37: venusian
```

Reply:
Bert just replied with an even better solution which is the environment markers in the testing extra of setup.py.

Reply:
The suggestion by @bertjwregeer is good, but we want to make it an optional dependency on Python 3.7. I believe his solution will install it automatically.

Reply:
Only for the testing extra, which is opt-in for the tests which is only used by `[testenv] extras = testing`. Of course don't add it to the default requirements.